docs/en/ingest-management/quick-starts.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[fleet-elastic-agent-quick-start]]
+= Quick starts
+
+Want to get up and running with {fleet} and {agent} quickly? See the quick start
+guides.
+
+[id="fleet-quick-start"]
```

Comment:
The sections here will be removed after the topics are successfully building in the Observability Guide.